src/pudl/package_data/nrelatb/column_maps/data.csv

```diff
@@ -0,0 +1,21 @@
+year_index,2019,2020,2021,2022,2023
+report_year,atb_year,atb_year,atb_year,atb_year,atb_year
+core_metric_key,core_metric_key,core_metric_key,core_metric_key,core_metric_key,core_metric_key
+core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter
+core_metric_case,core_metric_case,core_metric_case,core_metric_case,core_metric_case,core_metric_case
+cost_recovery_period_years,crpyears,crpyears,crpyears,crpyears,crpyears
+technology_description,technology,technology,technology,technology,technology
+technology_alias,,,technology_alias,technology_alias,technology_alias
```

Comment:
the description and the alias fields were very similar in content and i could not find a reference to this field in the documentation.